### ISSUE TITLE
[TailwindCSS JIT] `maizzle build` task hangs indefinitely

### ISSUE THREAD

**Issue by Xeevis** [maizzle/framework#434]
- Maizzle Version: 3.4.0
- TailwindCSS Version: 2.1.1
- Node.js Version: 15.14.0

When JIT mode is enabled in `tailwind.config.js` then `maizzle build` will not terminate properly, it will hang until manual termination `CTRL + C`.

```js
/* tailwind.config.js */
module.exports = {
  mode: process.env.NODE_ENV === "local" ? "jit" : "aot", 
  purge: ["src/**/*.html"],
  darkMode: "media",
  (...)
```

**Comment by cossssmin** [comment]
I think this is caused by Tailwind's watcher for JIT, which we need to enable or otherwise CSS isn't generated:

https://github.com/maizzle/framework/blob/660db550c30191937a526cfbeee3737b075b88ea/src/generators/tailwindcss.js#L16-L18

Disabling that will have `maizzle build` exit as expected, but it'll render JIT useless when using `maizzle serve` since no classes are generated (because it no longer watches the files you make changes to). 

One-off builds with `TAILWIND_MODE= 'build'` can't work here, because CSS will be generated only once when you run the command - subsequent changes to a file won't do anything.

To get around this, we'd need to enable `TAILWIND_MODE = 'watch'` only when running the `maizzle serve` command.

**Comment by cossssmin** [comment]
Fixed in v3.4.2

https://github.com/maizzle/framework/releases/tag/v3.4.2
